doc/cephadm/services/mgmt-gateway.rst

```diff
@@ -0,0 +1,136 @@
+.. _deploy-cephadm-mgmt-gateway:
+
+=============
+Mgmt Gateway
+=============
+
+Deploying mgmt-gateway
+======================
+
+Starting from version 19.x.x, the `mgmt-gateway` service introduces a new design for Ceph applications
+based on a modular, service-based architecture. This service, managed by cephadm and built on top of nginx
+(an open-source, high-performance web server), acts as the new front-end and single entry point to the
+Ceph cluster. The `mgmt-gateway` provides unified access to all Ceph applications, including the ceph dashboard
+and monitoring stack. Utilizing nginx enhances security and simplifies access management due to its robust
+community support and high-security standards. The `mgmt-gateway` service acts as a reverse-proxy that routes
+requests to the appropriate ceph backend application.
+
+In order to deploy the mgmt-gateway service, use the following command:
+
+.. prompt:: bash #
+
+    ceph orch apply mgmt-gateway [--placement ...] ...
+
+Once applied cephadm will automatically reconfigure several running daemons (including monitoring)
+to run automatically behind the new created service. External access to those services will not be possible
+anymore. All the access will be centralized through the new created service endpoint: `https://<node-ip>:<port>`.
+
+
+Benefits of the mgmt-gateway service
+====================================
+* ``Unified Access``: Centralizing access through nginx improves security and provide a single entry to the cluster.
+* ``Improved user experience``: User shouldn't care anymore about where each application is running (ip/host).
+* ``High Availability for dashboard``: nginx HA mechanisms are used to provide high availability for ceph dashboard.
+* ``High Availability for monitoring``: nginx HA mechanisms are used to provide high availability for monitoring.
```

Comment:
Do they prevent operators from detecting that an instance is down, or that the number of running instances is less than desired?